src/test/java/org/fundacionjala/at15/katas/banckorc/validAccountTest.java

```diff
@@ -0,0 +1,65 @@
+package org.fundacionjala.at15.katas.banckorc;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.fundacionjala.at15.katas.bankocr.ValidAccount;
+import org.junit.Test;
+
+public class validAccountTest {
+
+    @Test
+    public void itShouldBeValidAccount() {
+        boolean valid = true;
+        ValidAccount vAccount = new ValidAccount("D:\\workspace\\coding\\AT15\\src\\main\\java\\org\\fundacionjala\\at15\\katas\\bankocr\\txtFile\\validAccount.txt");
+        assertEquals(valid, vAccount.getValidation());
+    }
+
+    @Test
+    public void shouldBeAccountsToValidate() {
+        List<String> exp = new ArrayList<>();
+        //added for the test
+        exp.add("345882865");
+        exp.add("000000051");
+        //changed for the test
+        exp.add("49006771x");
+        //changed for the test
+        exp.add("12345678x");
+        ValidAccount vAccount = new ValidAccount("D:\\workspace\\coding\\AT15\\src\\main\\java\\org\\fundacionjala\\at15\\katas\\bankocr\\txtFile\\validAccount.txt");
+        assertEquals(exp, vAccount.getAccountsToValidate());
+    }
+
+    @Test
+    public void itShouldBeIntegers() {
+        boolean valid = true;
+        ValidAccount vAccount = new ValidAccount("D:\\workspace\\coding\\AT15\\src\\main\\java\\org\\fundacionjala\\at15\\katas\\bankocr\\txtFile\\validAccount.txt");
```

Comment:
When adding paths that reference files, you should use relative paths in the project instead of paths that reference to files in your machine, this way, your test will always run correctly in any environment (your machine, github, etc)

Reply:
For example
```
ValidAccount vAccount = new ValidAccount("src/main/java/org/fundacionjala/at15/katas/bankocr/txtFile/validAccount.txt")
```
Note that full path is not required (drive D:\ and path to project is not needed), assume project folder as the root of files. Also, normally the no-code files like .txt would be placed under src/main/resources folder of the project